packages/amplify_auth_cognito/example/lib/Widgets/ConfirmSignInWidget.dart

```diff
@@ -24,8 +24,8 @@ class _ConfirmSignInWidgetState extends State<ConfirmSignInWidget> {
     try {
       var res = await Amplify.Auth.confirmSignIn(
           confirmationValue: confirmationCodeController.text.trim());
-      widget.showResult('Confirm Sign In Status = ' + res.nextStep.signInStep);
-      widget.changeDisplay(res.nextStep.signInStep == 'DONE'
+      widget.showResult('Confirm Sign In Status = ${res.nextStep?.signInStep}');
+      widget.changeDisplay(res.nextStep!.signInStep == 'DONE'
```

Comment:
Sorry I just noticed this but we're using a `?` on one line but `!` casting on the other. Should line 28 be `?` also?

Reply:
yes it should be thanks for catching that.